.github/workflows/maven.yml

```diff
@@ -11,29 +11,38 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
 
     steps:
-      - name: Checkout sources
-        uses: actions/checkout@v1
-
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
-        with:
-          java-version: 11
-
-      - name: Build with Maven
-        if: matrix.os == 'ubuntu-latest'
-        run: mvn --batch-mode -Pjacoco install
-
-      - name: Build with Maven
-        if: matrix.os != 'ubuntu-latest'
-        run: mvn --batch-mode install
-
-      - name: Run SonarCloud analysis
-        if: matrix.os == 'ubuntu-latest'
-        run: >
-          mvn --batch-mode -DskipTests sonar:sonar
-          -Dsonar.host.url=https://sonarcloud.io
-          -Dsonar.organization=powsybl-ci-github
-          -Dsonar.projectKey=com.powsybl:powsybl-balances-adjustment
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+    - name: Checkout sources
+      uses: actions/checkout@v1
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+
+    - name: Checkout powsybl-core
+      uses: actions/checkout@v1
+      with:
+        repository: powsybl/powsybl-core
+          ref: refs/heads/emf-test
+
+    - name: Build and install powsybl-core with Maven
+      run: mvn --batch-mode -DskipTests=true --file ../powsybl-core/pom.xml install
+
+    - name: Build with Maven
+      if: matrix.os == 'ubuntu-latest'
+      run: mvn --batch-mode -Pjacoco install
+
+    - name: Build with Maven
+      if: matrix.os != 'ubuntu-latest'
+      run: mvn --batch-mode install
+
+    - name: Run SonarCloud analysis
+      if: matrix.os == 'ubuntu-latest'
+      run: >
+        mvn --batch-mode -DskipTests sonar:sonar
+        -Dsonar.host.url=https://sonarcloud.io
+        -Dsonar.organization=powsybl-ci-github
+        -Dsonar.projectKey=com.powsybl:powsybl-balances-adjustment
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
Oops, small problem of indent. Could you please correct it as it prevents jobs to launch?